client/my-sites/domains/domain-management/transfer/transfer-page/index.tsx

```diff
@@ -89,22 +99,87 @@ const TransferPage = ( props: TransferPageProps ): JSX.Element => {
 					mainText={ __( 'Transfer this domain to any site you are an administrator on' ) }
 				/>
 			</Card>
+		);
+	};
+
+	const requestTransferCode = () => {
+		requestDomainTransferCodeOnly( selectedDomainName );
+	};
+
+	const renderAdvancedTransferOptions = () => {
+		// translators: domain transfer lock
+		const disabledLockLabel = __( 'Transfer lock off' );
+		// translators: domain transfer lock
+		const enabledLockLabel = __( 'Transfer lock on' );
+
+		const toggleLabel = ! domain ? (
+			'loading'
+		) : (
+			<span className="transfer-page__transfer-lock-label">
+				{ domain.isLocked ? (
+					<>
+						<Icon icon={ lock } size={ 15 } viewBox="4 0 18 20" />
+						{ enabledLockLabel }
+					</>
+				) : (
+					disabledLockLabel
+				) }
+			</span>
+		);
+
+		return (
+			<Card>
+				<CardHeading size={ 16 } isBold={ true }>
+					Advanced Options
+				</CardHeading>
+				<ToggleControl
+					checked={ domain?.isLocked }
+					disabled={ false }
+					onChange={ () => null }
+					label={ toggleLabel }
+				/>
+				<p>
+					{ __(
+						'We recommend leaving the transfer lock on, unless you want to transfer your domain to another provider.'
+					) }
+				</p>
+				<Button primary={ false } busy={ isRequestingTransferCode } onClick={ requestTransferCode }>
+					{ __( 'Get authorization code' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **14** times:
`translate( 'Get domain authorization code' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/advanced-transfer-options-component)